### PR TITLE
Adjust link type wording (follow up for #3191)

### PR DESCRIPTION
**Pull request by MizukiTemma**
### Short description

This PR fixes wording of link type names which were not covered by #3191 

### Proposed changes

- Search & replace

### Side effects

- I'm not sure whether all targets are covered, would appreciate if reviewers too search for potential missed targets to replace actively.
- No release note is provided, as #3191 is not delivered to users and this is something additional to it.

### Resolved issues

Fixes: #3254 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/locale/nl/LC_MESSAGES/django.po
Comment (juliankehne) [inline integreat_cms/locale/nl/LC_MESSAGES/django.po:1]: Correct that its in the Dutch(nl) locale? Shouldn't it be English?

Comment (MizukiTemma) [reply integreat_cms/locale/nl/LC_MESSAGES/django.po:1]: its?

**Review by charludo — APPROVED** [review]
🙏🏼

**Review by JoeyStk — APPROVED** [review]
LGTM :)

**Pull request merged**